index.html

```diff
@@ -63,6 +71,9 @@ <h3>Written in python, best served as a bespoke,<br> high-performance e-commerce
           <source media="(max-width: 768px)" srcset="images/main-pic-mobile.svg">
           <img src="images/main-pic.svg" alt="Human">
         </picture>
+        <a href="https://github.com/mirumee/saleor">
+          <button class="btn btn-get-started">CLONE OR DOWNLOAD</button>
```

Comment:
Upper case letters should be done in CSS not in the html.

Reply:
It's done in the css too, but I'll change it to lowercase for the code consistency.